datasets/mkb/README.md

```diff
@@ -0,0 +1,163 @@
+---
+task_categories:
+- sequence-modeling
+multilinguality:
+- translation
+task_ids:
+- language-modeling
+languages:
+- Hindi
+- Telugu
+- Tamil
+- Malayalam
+- Gujarati
+- Urdu
+- Bengali
+- Oriya
+- Marathi
+- Punjabi
+- English
```

Comment:
Can you use the tagger app to fill the languages with the proper language codes ?
You can find more info here : https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card